src/node/building_blocks.ml

```diff
@@ -62,7 +68,7 @@ let is_signable state block =
     match operation with
     | Protocol.Operation.Core_tezos _ ->
       List.exists
-        (fun op -> Protocol.Operation.equal op operation)
+        (fun op -> Protocol.Operation.equal op.Node.operation operation)
```

Comment:
Why is this change necessary?

Reply:
Answer: Eduardo changed the type by wrapping it in a record that additionally includes the timestamp marking when it was requested.